cli/build-public-library.js

```diff
@@ -120,62 +105,51 @@ function processFiles(skyPagesConfig) {
   );
 }
 
-/**
- * Creates a UMD JavaScript bundle.
- * @param {*} skyPagesConfig
- * @param {*} webpack
- */
-function createBundle(skyPagesConfig, webpack) {
-  const webpackConfig = require('../config/webpack/build-public-library.webpack.config');
-  const config = webpackConfig.getWebpackConfig(skyPagesConfig);
-  return runCompiler(webpack, config);
-}
-
 /**
  * Transpiles TypeScript files into JavaScript files
  * to be included with the NPM package.
  */
 function transpile() {
-  return new Promise((resolve, reject) => {
-    const result = spawn.sync(
-      skyPagesConfigUtil.spaPath('node_modules', '.bin', 'ngc'),
-      [
-        '--project',
-        skyPagesConfigUtil.spaPathTemp('tsconfig.json')
-      ],
-      { stdio: 'inherit' }
-    );
-
-    // Catch ngc errors.
-    if (result.err) {
-      reject(result.err);
-      return;
-    }
+  const projectConfigPath = skyPagesConfigUtil.spaPathTemp('ng-package.json');
+  const tsConfigPath = skyPagesConfigUtil.spaPathTemp('tsconfig.json');
+
+  return ngPackage.ngPackagr()
```

Comment:
Normally this would be run with a CLI command (`ng-packagr -p package.json`), but we need to provide custom project and tsconfig files.